scraper/scraper.js

```diff
@@ -2,210 +2,348 @@ const axios = require("axios");
-    SZ: "swaziland",
-    SE: "sweden",
-    CH: "switzerland",
-    SY: "syria",
-    tibet: "tibet",
-    TR: "turkey",
-    UA: "ukraine",
-    GB: "united kingdom",
-    US: "united states",
-    wales: "wales",
-}
+  AF: "afghanistan",
+  AL: "albania",
+  AD: "andorra",
+  AR: "argentina",
+  AM: "armenia",
+  AU: "australia",
+  AT: "austria",
+  AZ: "azerbaijan",
+  BD: "bangladesh",
+  BY: "belarus",
+  BE: "belgium",
+  BR: "brazil",
+  BG: "bulgaria",
+  CA: "canada",
+  CL: "chile",
+  CN: "china",
+  "countries-throughout-world": "countries throughout the world",
+  HR: "croatia",
+  CY: "cyprus",
+  CZ: "czech republic",
+  DK: "denmark",
+  "eastern-mediterranean": "eastern mediterranean",
+  EG: "egypt",
+  england: "england",
+  FI: "finland",
+  FR: "france",
+  GE: "georgia",
+  DE: "germany",
+  "great-britain": "great britain",
+  GR: "greece",
+  holland: "holland",
+  HU: "hungary",
+  IS: "iceland",
+  IN: "india",
+  IR: "iran",
+  IQ: "iraq",
+  IE: "ireland",
+  IM: "isle of man",
+  IL: "israel",
+  IT: "italy",
+  JP: "japan",
+  JO: "jordan",
+  LB: "lebanon",
+  LT: "lithuania",
+  MK: "macedonia",
+  MR: "mauritania",
+  MX: "mexico",
+  "mexico-and-caribbean": "mexico and caribbean",
+  "middle-east": "middle east",
+  MN: "mongolia",
+  NP: "nepal",
+  NL: "netherlands",
+  NZ: "new zealand",
+  NO: "norway",
+  PK: "pakistan",
+  PS: "palestine",
+  PL: "poland",
+  PT: "portugal",
+  RO: "romania",
+  RU: "russia",
+  scotland: "scotland",
+  RS: "serbia",
+  SK: "slovakia",
+  ES: "spain",
+  SZ: "swaziland",
+  SE: "sweden",
+  CH: "switzerland",
+  SY: "syria",
+  tibet: "tibet",
+  TR: "turkey",
+  UA: "ukraine",
+  GB: "united kingdom",
+  US: "united states",
+  wales: "wales",
+};
 
 async function getCheeseInfo(link) {
-    return axios.get(link)
-        .then((response) => {
-            const dom = new JSDOM(response.data);
-            const { document } = dom.window;
-            const attrib_arr = Array.from(document.getElementsByClassName("summary-points")[0].getElementsByTagName("p"));
-
-            const attribs = {
-                made: null,
-                countries: [], // "and" ","
-                region: null,
-                family: null,
-                types: [], // ,
-                fat: null,
-                calcium: null,
-                textures: [], // "and" ","
-                rind: null,
-                color: null,
-                flavors: [], // ,
-                aromas: [], // ,
-                vegetarian: null,
-                producers: [], // ,
-                synonyms: [], // ,
-                alternative_spellings: [], // ,
-            }
+  return axios
+    .get(link)
+    .then((response) => {
+      const dom = new JSDOM(response.data);
+      const { document } = dom.window;
+      const attrib_arr = Array.from(
+        document
+          .getElementsByClassName("summary-points")[0]
+          .getElementsByTagName("p")
+      );
```

Comment:
big nitpick here but why is it broken down into 4 lines

Reply:
That's right, you like messy hard to read code, I forgot. Smh, ill fix tomorrow